projects/demo/src/app/demo/pages/form-control-page/state-example/form-control-state-example.component.html

```diff
@@ -2,11 +2,11 @@
 <p><strong>Important</strong>: <code>disabled</code> can not be used in combination with <code>FormControl</code> or <code>FormControlName</code> directive.</p>
 <pre><code>{{code}}</code></pre>
 <div>
-    <input paFormControl [disabled]="disabled" [readonly]="readonly">
+    <input paFormControl [attr.disabled]="disabled || undefined" [attr.readonly]="readonly || undefined">
```

Comment:
oh ok, so `[disabled` works on `pa-input` but not on native inputs, sorry I should have seen this!